File: PyGame/Entities/ScenarioInfo.py
from PyGame.Entities.Entity import Pedestrian, Vehicle
from PyGame.Entities.Environment import Environment

class ScenarioInfo:
    def __init__(self, name: str, mapURl: str, env: Environment, pedestrians: list[Pedestrian], vehicles: list[Vehicle],
                 egoVehicle: Vehicle):
        self.egoVehicle: Vehicle = egoVehicle
        self.vehicles: list[Vehicle] = vehicles
        self.pedestrians: list[Pedestrian] = pedestrians
        self.env: Environment = env
        self.name: str = name
        self.mapURL: str = mapURl

File: C#/ExportScenario/ExportScenario/Entities/ScenarioInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ExportScenario.Entities
{
    public class ScenarioInfo
    /// <summary>Create ScenarioInfo Obejct. Contains all GUI-Userinputs</summary>
    {
        public ScenarioInfo()
        {
        }

        public ScenarioInfo(string name, List<Pedestrian> pedestrians, string mapURL, WorldOptions worldOptions, Ego egoVehicle, List<Vehicle> vehicles)
        {
            Name = name;
            Pedestrians = pedestrians;
            MapURL = mapURL;
            WorldOptions = worldOptions;
            EgoVehicle = egoVehicle;
            Vehicles = vehicles;
        }

        public string Name { get; set; }
        public List<Pedestrian> Pedestrians { get; set; }
        public string MapURL { get; set; }
        public WorldOptions WorldOptions { get; set; }
        public Ego EgoVehicle { get; set; }
        public List<Vehicle> Vehicles { get; set; }

    }
}
